test/new_relic/agent/llm/feedback_test.rb

```diff
@@ -6,5 +6,60 @@
 
 module NewRelic::Agent::Llm
   class FeedbackTest < Minitest::Test
+    def setup
+      NewRelic::Agent.drop_buffered_data
+    end
+
+    def test_record_llm_feedback_event_records_required_attributes
+      in_transaction do |txn|
+        NewRelic::Agent.record_llm_feedback_event(trace_id: '01234567890', rating: 5)
+        _, events = NewRelic::Agent.agent.custom_event_aggregator.harvest!
+        type, attributes = events[0]
+
+        assert_equal 'LlmFeedbackMessage', type['type']
+        assert_equal '01234567890', attributes['trace_id']
+        assert_equal 5, attributes['rating']
+        assert_equal 'Ruby', attributes['ingest_source']
+        assert attributes['id']
+      end
+    end
+
+    def test_record_llm_feedback_event_records_optional_attributes
+      in_transaction do |txn|
+        NewRelic::Agent.record_llm_feedback_event(trace_id: '01234567890', rating: 5,
+          category: 'Helpful', message: 'Looks good!', metadata: {'pop' => 'tart', 'toaster' => 'strudel'})
+        _, events = NewRelic::Agent.agent.custom_event_aggregator.harvest!
+        type, attributes = events[0]
+
+        assert_equal 'Helpful', attributes['category']
+        assert_equal 'Looks good!', attributes['message']
+        assert_equal 'tart', attributes['pop']
+        assert_equal 'strudel', attributes['toaster']
+      end
+    end
+
+    def test_record_llm_feedback_event_raises_missing_required_parameter
+      assert_raises(ArgumentError) { NewRelic::Agent.record_llm_feedback_event(trace_id: '01234567890') }
+    end
+
+    def test_record_llm_feedback_event_invalid_param
+      assert_raises(ArgumentError) { NewRelic::Agent.record_llm_feedback_event(trace_id: '01234567890', rating: 5, food: 'blueberry') }
```

Comment:
This line is too long, but the linter stage of the automated CI process doesn't ding you on it because we've relaxed so many rules to work with the 15 year old code base.

@kaylareopelle and I were chatting about this recently. We'd like to come up with a solution for holding new code accountable to our desired RuboCop configuration while allowing the old code to be excepted. Perhaps a starting point would be to have new code forbidden from using the `.rubocop_todo.yml` file, though long term there are a whole heck of lot of exceptions expressed in `.rubocop.yml` too.

But for now for this PR, would you please get these lines at or under 120 columns in width? Thank you.

Reply:
Thank you for this callout! I never re-added a vertical line helper to my editor. Thats been done, and this code cleaned up. https://github.com/newrelic/newrelic-ruby-agent/pull/2468/commits/ce0ee2d8818c149a649fda49aebc8cc60615f3c9#diff-bc386c8b22f64519a262c1bde1fb6b92218a86c109d9da8df409a91a0bb5be08R47-R52